workers/redirector.js

```diff
@@ -26,6 +26,26 @@ const experimentPages = [
         'sandboxPath': `/en-US/exp/firefox/`,
         'sampleRate': 0.34
     },
+    {
+        'targetPath': `/en-US/firefox/accounts/`,
+        'sandboxPath': `/en-US/exp/firefox/accounts/`,
+        'sampleRate': 0.52
+    },
+    {
+        'targetPath': `/fr/firefox/`,
+        'sandboxPath': `/fr/exp/firefox/`,
+        'sampleRate': 0.51
+    },
+    {
+        'targetPath': `/de/firefox/`,
+        'sandboxPath': `/de/exp/firefox/`,
+        'sampleRate': 0.68
+    },
+    {
+        'targetPath': `/de/firefox/new`,
```

Comment:
I think the `*` in paths should get us to the worker, but I'm not 100% sure. Reading the cloudflare docs doesn't make it obvious to me how it ought to work.  I think the easiest thing to do is just merge it, and give it a try.

Reply:
sounds good!